wiki/Ranking_Criteria/en.md

```diff
@@ -60,7 +60,7 @@ Important to understand before using:
   - **...lower than 3:30**, the lowest difficulty of each included game mode cannot be harder than a Normal.
   - **...between 3:30 and 4:15**, the lowest difficulty of each included game mode cannot be harder than a Hard.
   - **...between 4:15 and 5:00**, the lowest difficulty of each included game mode cannot be harder than an Insane.
-  - **Difficulties below the highest difficulty can combine break times with drain time to meet the above thresholds.** This does not apply to difficulties with less than 30 seconds of drain time. If the lowest difficulty of the beatmap is a Hard or Insane, judgement regarding the suitability of its gameplay elements is up to the [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) and [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) members for the corresponding game modes.
+  - **[Break times](/wiki/Beatmap/Break) may be combined with [drain time](/wiki/Beatmap/Drain_time) to meet the above thresholds.** For the highest difficulty, this is limited to at most 30 seconds of break time. This does not apply to difficulties with less than 30 seconds of drain time.
```

Comment:
This latest version still doesn't address my concerns [here](https://github.com/ppy/osu-wiki/pull/8861#discussion_r1119932473). The wording of this rule means it would be valid to make an easy difficulty with 30s drain time and 5 minutes break time.

Reply:
it has always been like this and has never been an issue, so I'm afraid that's out of scope for now

Reply:
Okay. I'm still very unhappy with the wording but if you guys think this is the correct path then let's see.